sympy/geometry/polygon.py

```diff
@@ -2270,6 +2270,69 @@ def exradii(self):
         return exradii
 
     @property
+    def excenters(self):
+        """Excenters of the triangle.
+
+        An excenter is a point at which the line bisecting one interior angle meets the
+        bisectors of the two exterior angles on the opposite side. This point is the center
+        of excircle. Every triangle has three excenters and three excircles.
+
+        Returns
+        =======
+
+        excenters : dict
+
+
+        Examples
+        ========
+
+        The center of excircle is called the excenter relative to the vertex. The center is
```

Comment:
Or, perhaps,

The excenters are keyed to the side of the triangle to which their corresponding excircle is tangent:

Reply:
Sure. I used textbook definition but for the sake of honesty, I'd prefer more clear definition.